src/contexts/CartContext/CartContext.tsx

```diff
@@ -0,0 +1,79 @@
+
```

Comment:
See Kent's article for better patterns for this file: https://kentcdodds.com/blog/how-to-use-react-context-effectively#typescript